addon/webextension/background/analytics.js

```diff
@@ -129,24 +129,24 @@ this.analytics = (function() {
   };
 
   exports.refreshTelemetryPref = function() {
-    return communication.sendToBootstrap("getTelemetryPref").then((result) => {
+    return communication.sendToBootstrap("isTelemetryEnabled").then((result) => {
       telemetryPrefKnown = true;
       if (result === communication.NO_BOOTSTRAP) {
-        telemetryPref = true;
+        telemetryEnabled = true;
       } else {
-        telemetryPref = result;
+        telemetryEnabled = result;
       }
     }, (error) => {
       // If there's an error reading the pref, we should assume that we shouldn't send data
       telemetryPrefKnown = true;
-      telemetryPref = false;
+      telemetryEnabled = false;
       throw error;
     });
   };
 
-  exports.getTelemetryPrefSync = function() {
+  exports.isTelemetryEnabled = function() {
     catcher.watchPromise(exports.refreshTelemetryPref());
-    return !!telemetryPref;
+    return !!telemetryEnabled;
```

Comment:
Nit (and not even part of these changes): if `telemetryEnabled` defaults to false, then we don't need the `!!` here.